bin/rails

```diff
@@ -1,9 +1,5 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path('../spring', __FILE__)
-rescue LoadError => e
```

Comment:
is there a reason to remove the rescue ?

Reply:
no, that is a default file.